sphinxcontrib/katex.py

```diff
@@ -133,13 +132,18 @@ def copy_katex_css_file(app, css_file_name):
 
 
 def katex_autorenderer_content(app):
-    content = 'renderMathInElement(document.body, latex_options);'
+    content = '''
+document.addEventListener("DOMContentLoaded", function() {
+  renderMathInElement(document.body, katex_options);
+});
+'''
+    prefix = 'katex_options = {'
+    suffix = '}'
     macros = app.config.katex_macros
     if len(macros) > 0:
-        prefix = 'latex_options = { macros: {'
-        suffix = '}}'
-        content = '\n'.join([prefix, macros, suffix, content])
-    return content
+        macros = 'macros: {' + macros + '},'
+    options = app.config.katex_options
+    return '\n'.join([prefix, macros, options, suffix, content])
```

Comment:
It's actually a good idea to allow for [all available KaTeX options](https://github.com/Khan/KaTeX#rendering-options). To make it more consistent we should then drop `katex_macros` as an extra option and handle everything under `katex_options`.
What do you think?

Reply:
I kept the `katex_macros` for backwards compatibility, but if you think it can be deleted, that's fine too. Maybe we can check and only allow one or the other.

Reply:
That's a good point. As this project is quite new it should be ok to break backward compatibility. But checking for it might be a good solution. Maybe I first integrate your current solution and change it afterwards.

Reply:
OK I changed the URL lines, so I'll let you handle this as you see fit.